### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by igor-stasiv**
[DEMO LINK](https://igor-stasiv.github.io/react_add-todo-form/)

### CODE REVIEW

File: src/components/TodoList/TodoList.jsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+      }
+
+      if (!title || title.length > 35 || title.length < 5) {
+        return { isTitleAdd: false };
+      }
+
+      if (!userName) {
+        return { isUserSelected: false };
+      }
+
```
Comment (Ihor-Karpyn) [inline src/components/TodoList/TodoList.jsx:54]: Maybe move these logic to handlers

**Review by Ihor-Karpyn — APPROVED** [review]

File: src/components/TodoList/TodoList.jsx (hunk 1 above)
Comment (varseniuk) [reply src/components/TodoList/TodoList.jsx:54]: Well, these checks should be triggered on form submit, so let them be here

File: src/components/TodoList/TodoList.jsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+      return ({
+        todos: [...todos, {
+          id: todos[todos.length - 1].id + 1,
```
Comment (varseniuk) [inline src/components/TodoList/TodoList.jsx:57]:
```suggestion
          id: todos.length + 1,
```

**Review by varseniuk — APPROVED** [review]
Really good job! 🔥

**Pull request closed without merging**